.github/workflows/dlp-pipelines.yml

```diff
@@ -0,0 +1,270 @@
+name: Java CI with Gradle
+
+on:
+#  pull_request:
+#    types: [ opened, reopened, synchronize ]
+  schedule:
+    - cron: '30 9 * * *'
+
+env:
+  PROJECT_ID: "dlp-dataflow-deid-ci-392604"
+  DATASET_ID: "demo_dataset"
+  GCS_BUCKET: "dlp-dataflow-deid-ci-392604-demo-data"
+  INPUT_FILE_NAME: "tiny_csv"
+  INSPECT_TEMPLATE_PATH: "projects/dlp-dataflow-deid-ci-392604/locations/global/inspectTemplates/dlp-demo-inspect-latest-1689137435622"
+  DEID_TEMPLATE_PATH: "projects/dlp-dataflow-deid-ci-392604/locations/global/deidentifyTemplates/dlp-demo-deid-latest-1689137435622"
+  REID_TEMPLATE_PATH: "projects/dlp-dataflow-deid-ci-392604/locations/global/deidentifyTemplates/dlp-demo-reid-latest-1689137435622"
+  SERVICE_ACCOUNT_EMAIL: "demo-service-account@dlp-dataflow-deid-ci-392604.iam.gserviceaccount.com"
+  PUBSUB_TOPIC_NAME: "demo-topic"
+  INSPECTION_TABLE_ID: "dlp_inspection_result"
+  NUM_INSPECTION_RECORDS: "30"
+  REIDENTIFICATION_QUERY_FILE: "reid_query.sql"
+
+jobs:
+  inspection:
+    runs-on:
+      - self-hosted
+      - test-runner-chitara01
```

Comment:
Please rename to something team specific?

Reply:
These are testing tags, have removed them.